bonus_material/GPT_neo_IMDB_multinode/GPT-neo-IMDB-finetuning.py

```diff
@@ -0,0 +1,302 @@
+#!/usr/bin/env python
```

Comment:
I guess this is meant as a helping hand to Samuel for the lecture 10 exercises? At least I believe that is should be part of the lecture 10 exercises and not included as bonus material.